plugins/janus_streaming.c

```diff
@@ -79,11 +79,14 @@ videobufferkf = yes|no (whether the plugin should store the latest
 	keyframe and send it immediately for new viewers, EXPERIMENTAL)
 databuffermsg = yes|no (whether the plugin should store the latest
 	message and send it immediately for new viewers)
+x-audio-multicast-iface = network interface IP address or device name to listen on when receiving multicast audio
+x-video-multicast-iface = network interface IP address or device name to listen on when receiving multicast video
```

Comment:
OK, I'll change the name into something more uniform with existing config keys. Currently it does not make sense in a unicast context because the code explicitly uses these interfaces to populate the structuring describing the multicast in the `IP_ADD_MEMBERSHIP` operation.

I do not know if this could possibly make (any) sense for unicast  addresses: as I understand it, unicast addresses are bound to a particular interface anyway by definition.